src/main/java/io/github/thebusybiscuit/slimefun4/utils/ColoredMaterial.java

```diff
@@ -199,29 +199,29 @@
             Material.GREEN_SHULKER_BOX,
             Material.RED_SHULKER_BOX,
             Material.BLACK_SHULKER_BOX
-    }),
+    });
     
     /**
      * This {@link List} contains all candle colors ordered by their appearance ingame.
      */
-    CANDLE(new Material[] {
-            Material.WHITE_CANDLE,
-            Material.ORANGE_CANDLE,
-            Material.MAGENTA_CANDLE,
-            Material.LIGHT_BLUE_CANDLE,
-            Material.YELLOW_CANDLE,
-            Material.LIME_CANDLE,
-            Material.PINK_CANDLE,
-            Material.GRAY_CANDLE,
-            Material.LIGHT_GRAY_CANDLE,
-            Material.CYAN_CANDLE,
-            Material.PURPLE_CANDLE,
-            Material.BLUE_CANDLE,
-            Material.BROWN_CANDLE,
-            Material.GREEN_CANDLE,
-            Material.RED_CANDLE,
-            Material.BLACK_CANDLE
-    });
+//    CANDLE(new Material[] {
+//            Material.WHITE_CANDLE,
+//            Material.ORANGE_CANDLE,
+//            Material.MAGENTA_CANDLE,
+//            Material.LIGHT_BLUE_CANDLE,
+//            Material.YELLOW_CANDLE,
+//            Material.LIME_CANDLE,
+//            Material.PINK_CANDLE,
+//            Material.GRAY_CANDLE,
+//            Material.LIGHT_GRAY_CANDLE,
+//            Material.CYAN_CANDLE,
+//            Material.PURPLE_CANDLE,
+//            Material.BLUE_CANDLE,
+//            Material.BROWN_CANDLE,
+//            Material.GREEN_CANDLE,
+//            Material.RED_CANDLE,
+//            Material.BLACK_CANDLE
+//    });
```

Comment:
why is this commented out? what's the alternative?

Reply:
there isnt yet

This was for testing

Candles will be turned into tags aswell
Will do after i get feedback that something like this actually gets accepted

Reply:
Don't see why we wouldn't. It is a breaking API change though.

Please cleanup your git history.